tests/test_martingale.py

```diff
@@ -0,0 +1,50 @@
+from unittest import TestCase
+
+from unittest.mock import Mock, patch
+
+from roulette import Game, Martingale, Table, Wheel, Bet, BinBuilder
+
+
+class TestMartingale(TestCase):
+    def setUp(self):
+        self.table = Table()
+        self.wheel = Wheel()
+        bin_builder = BinBuilder()
+        bin_builder.buildBins(self.wheel)
+        self.game = Game(self.wheel, self.table)
+        self.martingale = Martingale(self.table)
+
+    def test_placing_bets_need_minimum_stake(self):
```

Comment:
better rename to `test_bets_placed_when_stake_not_less_than_betMultiple`

Reply:
Done.